### DISCUSSION TITLE
I'm unable to get Achievements ?

### DISCUSSION THREAD

**Discussion by UmerGillani36** [community/community#55498] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

=> I'm the first to open the pull request and merge multiple branches but i don't get the pull shark.
=> I had merge the contributed branch but haven't get any achievement.

**Comment by ryanlua** [comment]
Hi @UmerGillani36,

Can you make sure those pull requests and branches that you merged are from a public repository?

You need 2 pull requests for the pull shark achievement, and not one.

**Reply by UmerGillani36** [reply]
My friend get it on private repo.

**Reply by ryanlua** [reply, marked as answer]
> My friend get it on private repo.

Can you go to your GitHub profile > Contribution settings and make sure "Private contributions" is enabled.

From your GitHub profile, it seems you only have 1 merged pull request for me.

**Reply by UmerGillani36** [reply]
yeah because one merge is in public repo and i have merged multiple branches in private repo, and how i can show my organization progress in my profile

**Reply by UmerGillani36** [reply]
Done thank you
